lib/engines/hckinstall/kits/HLK11_23H2.json

```diff
@@ -1,5 +1,4 @@
 {
-  "download_url": "https://go.microsoft.com/fwlink/?linkid=2250318",
```

Comment:
Are 23H2 version no longer supports by Microsoft?

Reply:
Looks like for now